repose-aggregator/core/repose-core-api/src/main/scala/org/openrepose/core/filter/AbstractConfiguredFilter.scala

```diff
@@ -119,14 +123,31 @@ abstract class AbstractConfiguredFilter[T: ClassTag](val configurationService: C
       doWork(request, response, chain)
       logger.trace("{} returning response...", this.getClass.getSimpleName)
     }
-
   }
 
   /**
     * Where the concrete class does it's work. This method is the equivalent doFilter in a normal filter.
+    *
     * @param request
     * @param response
     * @param chain
     */
   def doWork(request: ServletRequest, response: ServletResponse, chain: FilterChain): Unit
+
+  /**
+    * Unsubscribes from the configuration service.
+    */
+  override def destroy(): Unit = {
+    logger.trace("{} destroying ...", this.getClass.getSimpleName)
+    configurationService.unsubscribeFrom(configFile, this)
+    doDestroy()
+    logger.trace("{} destroyed.", this.getClass.getSimpleName)
+  }
+
+  /**
+    * Called immediately after the configFile is unsubscribed from.
+    */
+  def doDestroy(): Unit = {
+    logger.trace("{} Default postDestroy ...", this.getClass.getSimpleName)
```

Comment:
Right, but does anyone actually care if the default is called, considering it doesn't do anything?

Also consider that if a subclass did override this method, that subclass could actually log the exact same message using the exact same logger. So having this message in the logs is not strictly proof that a subclass did not override this method.